tldw_Server_API/tests/Claims/test_claims_rebuild_service_failure.py

```
@@ -25,3 +27,160 @@ def _boom(task: ClaimsRebuildTask):  # noqa: ARG001
         assert stats.get("processed", 0) == 0
     finally:
         svc.stop()
+
+
+def test_claims_rebuild_service_persist_health_uses_managed_media_database(monkeypatch):
+    class _FakeDb:
+        def __init__(self) -> None:
+            self.health_calls: list[dict[str, object]] = []
+
+        def upsert_claims_monitoring_health(self, **kwargs) -> None:
```

Ruff (0.15.6)

[warning] Missing type annotation for `**kwargs`

(ANN003)